database/migrations/default/1728407832045_engineering_areas_edits/down.sql

```diff
@@ -0,0 +1,91 @@
+--revert to name updated_at
+alter table engineering_areas rename column created_at to updated_at;
+--revert time to timestamp with out time zone
+alter table engineering_areas alter column updated_at set data type timestamp using updated_at::timestamp;
+-- rever other col names
+alter table engineering_areas rename column engineering_area_id to area_id;
+alter table engineering_areas rename column atd_engineer_areas to label;
```

Comment:
is this to match the col names from arcgis?

Reply:
yes—it's an unfortunate name 😞 

i'm obviously leaning in favor of making all the layers mirror their AGOL counterparts as much as possible, but i don't feel super strongly about it.